activemq-artemis/templates/master-service.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   name: {{ include "artemis.fullname" . }}-master
   labels:
 {{ include "artemis.labels.standard" . | indent 4 }}
+    component: artemis-node
```

Comment:
I wonder (just wonder because I really don't have an answer) if it would make sense to prefix the label the with helm release? Is this fully static for monitoring purposes?

Reply:
This one is interesting :) Its purpose is to act as a catch for the Prometheus ServiceMonitor. Now, ideally we would only need ONE ServiceMonitor regardless of how many releases we had in the cluster. So it makes sense to keep the label static, and have the ServiceMonitor collect from all deployed instances, regardless of release.

However (!) - I think if we tried to deploy the same chart twice, the fact that the ServiceMonitor is named statically would mean that it might fail (already exists), or simply overwrite it (which is OK). I confess I haven't tried it to test that. If it fails (already exists), then we would need to make that ServiceMonitor name/job include the Release Name, in which case we would want the label to also be release-specific, so that each ServiceMonitor only scrapes the targets for it's own release.

So I think I need to test out a multiple install of the Helm chart and see what happens to the ServiceMonitor. If it overwrites, I think the above is OK. If it fails, I'll need to refactor. We don't personally deploy the release twice (we scale up/upgrade, rather than deploy multiple releases), but I think it's a valid use-case.

Reply:
I guess we can keep iterating, this doesn't have to be final.